test/network_autoyast_test.rb

```diff
@@ -353,7 +353,7 @@ def keep_install_network_value(value)
     before(:each) do
       allow(Yast::LanItems)
         .to receive(:Items)
-        .and_return(0 => { "ifcfg" => "eth0" })
+        .and_return(0 => { "ifcfg" => "eth0", "udev" => { "net" => ["ATTR{address}==\"24:be:05:ce:1e:91\"", "KERNEL==\"eth*\"", "NAME=\"eth0\""] } })
```

Comment:
Is *this* supposed to be a test for this PR?

What about a new test case that fails without the fix and succeeds with the fix?
What about summarizing the 62 Bugzilla comments so far so that a reviewer gets an idea of what the bug even is?

Reply:
this is necessary fix to keep the testsuite working after the patch. I'll add testsuite for new methods.